tests/suites/casper/viewport.js

```diff
@@ -16,6 +16,7 @@ casper.test.begin('viewport() tests', 3, function(test) {
 casper.test.begin('viewport() asynchronous tests', 2, function(test) {
     var screenshotData;
 
+    casper.options.pageSettings.javascriptEnabled = true;
```

Comment:
Spurious; as far as I remember this triggered a different problem (recursive error / warn calls).